src/abstract_oracles.jl

```diff
@@ -295,9 +315,9 @@ All keyword arguments are passed to all LMOs.
 """
 function compute_extreme_point(
     lmo::ProductLMO{N},
-    direction::AbstractArray;
+    direction::AbstractArray,
+    direction_indices;
```

Comment:
since this function has a signature different from `compute_extreme_point(lmo, direction)`, it would be better to use a different function name for it
`compute_extreme_point_at_indices(lmo, direction, indices)` for example